### PR TITLE
Add Community & Events into the footer

### PR DESCRIPTION
**Pull request by ahmadmoawad**
- Add Community & Events into the footer
- Implement new Footer design

`Large screen`

Screen Shot 2021-07-09 at 3 38 25 PM

---

`Medium screen`

Screen Shot 2021-07-12 at 1 33 25 PM

---

`Small screen`

Screen Shot 2021-07-12 at 1 33 39 PM

### CODE REVIEW

File: packages/dotcom-ui-footer/bower.json
Comment (ivomurrell) [inline packages/dotcom-ui-footer/bower.json:4]:
```suggestion
    "o-footer": "^9.0.0-bower"
```
**nitpick:** We probably want to use a [caret range](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept) so we can pick up any minor releases to `o-footer`

File: packages/dotcom-ui-footer/src/components/partials.tsx
Comment (maxcbc) [inline packages/dotcom-ui-footer/src/components/partials.tsx:67]: The className seems to have changed here, is this intentional?
It has gone from `o-footer__matrix-group o-footer__matrix-group--${submenu.length}` to `o-footer__matrix-group o-footer__matrix-group--1`

File: packages/dotcom-ui-footer/src/components/partials.tsx
Comment (maxcbc) [inline packages/dotcom-ui-footer/src/components/partials.tsx:77]: This id should be generated programmatically I think, as we don't know for sure it is number 5.
I think it should be `o-footer-section-${footerData.length}`

**Review by maxcbc — CHANGES_REQUESTED** [review]
There are a few formatting/white-space changes that we should probably avoid as it can make the git history confusing. Also there is a couple of hardcoded numeric ids that I think should be programmatically generated (but I could be wrong).

Other than that looks good. 🙂

**Review by ivomurrell — COMMENTED** [review]
**issue:** There seems to be some text overflow from one column to the next: 
FT Footer showing overflow from each column to the next, manifesting as text overlapping with other words
(Note the 'Today's Newspaper (ePaper)' for instance.)
**suggestion:** It looks like you can see some similar behaviour in the `o-footer` demo by resizing the window. Probably something that needs to be fixed upstream?

**Comment by ahmadmoawad** [comment]
> **issue:** There seems to be some text overflow from one column to the next:
> FT Footer showing overflow from each column to the next, manifesting as text overlapping with other words
> (Note the 'Today's Newspaper (ePaper)' for instance.)
> **suggestion:** It looks like you can see some similar behaviour in the `o-footer` demo by resizing the window. Probably something that needs to be fixed upstream?

Noted

**Comment by ahmadmoawad** [comment]
> **issue:** There seems to be some text overflow from one column to the next:
> FT Footer showing overflow from each column to the next, manifesting as text overlapping with other words
> (Note the 'Today's Newspaper (ePaper)' for instance.)
> **suggestion:** It looks like you can see some similar behaviour in the `o-footer` demo by resizing the window. Probably something that needs to be fixed upstream?

Hi @ivomurrell ,

The `o-footer` has been fixed and the new release now available`v9.0.1-bower`

**Review by emortong — APPROVED** [review]
LGTM, but I would wait for @ivomurrell 's approval as well before merging since he was reviewing it last week

File: packages/dotcom-ui-footer/bower.json
Comment (maxcbc) [reply packages/dotcom-ui-footer/bower.json]: Unfortunately, as o-footer isn't pushed to bower by default any more, we need to fix the version here. Otherwise `^9.0.0-bower` resolves to `9.0.0` and caused dependency issues due to origami moving to `npm`

Comment (ivomurrell) [reply packages/dotcom-ui-footer/bower.json]: Aaah that will be why I was having issues with storybook then! Looks to be fixed now, thanks.

**Review by ivomurrell — APPROVED** [review]
Footer looks good now, thanks!

**Review by maxcbc — APPROVED** [review]

**Pull request merged**
